internal_ws/ykpack/src/types.rs

```diff
@@ -362,12 +365,12 @@ impl Display for Ptr {
     }
 }
 
-/// An IR place. This is used in SIR and TIR to describe the (abstract) address of a piece of data.
+/// An IR place XXX why isn't the struct called "IRPlace"? [I've been wondering for ages what "IPlace" meant! "IRPlace" would be a bit clearer I think.]. This is used in SIR and TIR to describe the (abstract) address of a piece of data.
```

Comment:
No objections to renaming it. I think we inherited the name from Cranelift.

Reply:
Fixed in #263.